alg-excer1-palindrome.js

```diff
@@ -0,0 +1,25 @@
+function palindrome(input) {
+  var regExp = /[^A-za-z0-9]/g;
```

Comment:
Do not use anything you don't understand. How does regExp works? What is it doing? Do you know why it's inefficient? 

You should ALWAYS understand how something works or at least have any idea what it's doing before using it.

Reply:
used to replace non-alphanumeric characters
A-Z identifies anything that is not between A and Z
a-z identifies anything that is not between a and z
0-9 identifies anything that is not between 0 and 9